src/QueryStrategies/BatchQueryStrategies/AllRandomBatchQs.jl

```diff
@@ -0,0 +1,20 @@
+struct AllRandomBatchQs <: BatchPQs
+    k::Int
+
+    function AllRandomBatchQs(;k::Int=0)::AllRandomBatchQs
+        (k < 1) && throw(ArgumentError("Invalid batch size k=$(k)."))
+        return new(k)
+    end
+end
+
+"""
+Select batch by selecting batch_size indices from candidate_indices at random
+"""
+function select_batch(qs::AllRandomBatchQs, x::Array{T, 2}, labels::Dict{Symbol, Vector{Int}}, candidate_indices::Vector{Int})::Vector{Int} where T <: Real
+    num_observations = length(candidate_indices)
+    if num_observations <= qs.k
+        return candidate_indices
+    end
+
+    return sample(candidate_indices, qs.k, replace=false, ordered=true)
```

Comment:
We should never rely on the order of candidates, so the additional sorting by `ordered=true` should not be necessary.

Reply:
As far as I understand the [documentation](http://juliastats.github.io/StatsBase.jl/stable/sampling/#StatsBase.sample), the `ordered=true` argument only enforces that the returned array is ordered. So, there are no requirements on the order of the candidates.

I guess we can simply drop the argument because we do not care about the order of the chosen batch indices.